migration_scripts/twitter_author_membership.py

```diff
@@ -0,0 +1,61 @@
+"""Batch script for data migration on data source Twitter
+If the field author.created_at doesn't exist, fetch the information via the Tweepy API, calculate
+the field member_since based on author.created and insert_date / comment.created and add both
```

Comment:
There is no `comments` property for tweet documents in our db. Please update docstring accordingly